mangaki/mangaki/utils/stats.py

```diff
@@ -0,0 +1,7 @@
+import math
+
+def avgstd(l):  # Displays mean and variance
+    n = len(l)
+    mean = float(sum(l)) / n
+    var = float(sum(i * i for i in l)) / n - mean * mean
+    return '%.5f ± %.5f' % (round(mean, 5), round(1.96 * math.sqrt(var / n), 3))  # '%.3f ± %.3f' % 
```

Comment:
Why do you round to 3 decimals for the std part and then in the format, you round to 5 decimals?

I am not a floating point expert, but I'd round the values up to 6 digits and format them to 5 digits.
This way, you *might* avoid some floating point imprecisions.